docs/src/IASI.md

```diff
@@ -0,0 +1,208 @@
+## IASI
+
+The Infrared Atmospheric Sounding Interferometer (IASI) is an instrument on the METOP satellites. It is a hyper-spectral measuring the radiation from the atmosphere and earth in many spectral channels. These measurements are used to derive temperature and humidity profiles at high vertical resolution. This makes IASI an key data source for numerical weather prediction (NWP).
+See [ASI Level 1: Product Guide](https://user.eumetsat.int/s3/eup-strapi-media/pdf_iasi_pg_487c765315.pdf) and [IASI Level 2: Product Guide](https://user.eumetsat.int/s3/eup-strapi-media/IASI_Level_2_Product_Guide_8f61a2369f.pdf) for more information.
+
+
+## Static plot of L1C spectrums
+This example is made using the following packages.
+```
+[13f3f980] CairoMakie v0.12.16
+[db073c08] GeoMakie v0.7.9
+```
+
+The main key variable in the "gs1cspect" which contains the radiance spectrum measured by the IASI instrument. The spectrum from a full orbits is almost 2 GB of data. In this example we will just load one data record with observations locations and plot the spectrum of two observations. The spectrums are converted from radiances to brightness temperature since this is often most convenient when interpreting the IASI spectrums. The cloud cover is also read from the file and included as legends on the plot of the spectrums. Cloud cover is essential to understanding the spectrums and normally only cloud free observations are assimilated in NWP.  
```

Comment:
main key -> main (or key, but not both?)
variable in the -> variable is the
full orbits -> full orbit
observations locations -> observation locations
spectrums -> spectra (4 occurrences)